tools/virtualization/playing-around-with-various-vga-providers.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Playing around with various VGA providers  
+subsection: virtualization  
+order: 11  
+---
+
+# Playing around with various VGA providers
```

Comment:
This should have some more concise title like e.g.:

 ```suggestion
# Setting up various VGA providers
```

Reply:
The reason why I would suggest it to stay the same is because most VGA providers are very experimental in nature and people would not want to use them in a critical environment. They would still want to play around with them as these are actively developed and might be stable someday. What do you think?

Reply:
Maybe that could be mentioned stand alone / explicitly - maybe with Experimental in the sub-title? (Instead of being "in between the lines", as many non-native speakers won't capture that information.)

Reply:
How about "Experimenting with various VGA providers" as a title?

Reply:
Sounds great!